client/app/components/service-status/service-status-action.controller.js

```diff
@@ -0,0 +1,66 @@
+import {
+  DEFAULT_TARGET,
+  RENEW_URL,
+} from './service-status-action.constant';
+
+export default class {
+  constructor(constants, OvhApiMe) {
+    'ngInject';
```

Comment:
I think we prefere the `/* @ngInject */`, is there any selected way to do this @antleblanc ?

Reply:
@FredericEspiau is right.

```suggestion
    /* @ngInject */
    constructor(constants, OvhApiMe) {
```